### ISSUE TITLE
How can I disable animations

### ISSUE THREAD

**Issue by xseniSec** [andyhqtran/Divi-100-Article-Card#51]
### Problem:
I have problem with my personal site shinsec.pl
I dont want the effect of falling text after the page is loaded. How can I disable it?

This is happening on all pages, homepage, post page, etc.

Here is video of what i mean:
https://www.dropbox.com/s/exqej41byszt152/shin.mov?dl=0

### Steps To Reproduce:
Just load any page at my shinsec.pl

### Additional Information:

**Divi Version**:
3.0.7

**Comment by andyhqtran** [comment]
Does this issue happen when the plugin is disabled?

**Comment by xseniSec** [comment]
Yes

**Comment by andyhqtran** [comment]
Yes, but does this issue occur for you when the plugin is **disabled**?

**Comment by xseniSec** [comment]
Yes.

**Comment by andyhqtran** [comment]
I recommend reaching out to the support team if the jumping text you provided in this [link](https://www.dropbox.com/s/cqe2bytedtxdhvv/Video%2022-09-2016%2C%2000%2029%2045.mov?dl=0) causes any issue while the plugin is disable. This could be a core Divi bug instead a conflict with the plugin.
